### ISSUE TITLE
Any possibility of an option to just download ROMs/CHDs and leave them as is?

### ISSUE THREAD

**Issue by geoneil** [sam-ludlow/mame-ao#5]
This is amazing software.  I want to thank you for this.  I've never seen a more convenient way to just get and run games in MAME.  

However, I want to be able to launch my games from a different frontend (specifically Playnite) and use the ROMs in alternative emulators, examples...

- Running NAOMI, NAOMI2 and Atomiswave ROMs in Flycast
- Hikaru ROMs in DEmul (at least until a different emulator adds support for Hikaru)
- Lindbergh ROMs in Teknoparrot

Basically ROMs for other systems that run better in emulators other than MAME.  

Would you consider adding an option to just download the ROMs or export the ROMs to their original .zip format?  

Or is your software that good that I'm now too lazy to go straight to archive.org?!

**Comment by sam-ludlow** [comment]
Hello @geoneil I have added a .export command in 1.55.

It only does machine ROM at the moment, CHD & SL coming soon.

Cheers
Sam

**Comment by sam-ludlow** [comment]
Export is now fully operational in 1.56.

Cheers
Sam

**Comment by sam-ludlow** [comment]
Closing, import & export implemented.
